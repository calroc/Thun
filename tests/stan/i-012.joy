23 i
true i
