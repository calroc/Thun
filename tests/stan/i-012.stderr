Not a list.
Not a list.
